lib/flight_job/outputs/list_templates.rb

```diff
@@ -28,28 +28,27 @@
 require 'output_mode'
 
 module FlightJob
-  module Outputs::ListTemplates
-    extend OutputMode::TLDR::Index
-
-    register_column(header: 'Index', row_color: :yellow) do |template|
-      # NOTE: The OutputMode library does not support *_with_index type notation
-      #       Instead the index needs to be cached on the object itself
-      template.index
-    end
-    register_column(header: 'Name') do |template|
-      template.id
-    end
-    file_header = "File (Dir: #{FlightJob.config.templates_dir})"
-    register_column(header: file_header, verbose: true) do |template, interactive:|
-      if interactive
-        Pathname.new(template.workload_path).relative_path_from FlightJob.config.templates_dir
-      else
-        template.workload_path
+  class Outputs::ListTemplates < OutputMode::Formatters::Index
+    constructor do
+      register(header: 'Index', row_color: :yellow) do |template|
+        # NOTE: The OutputMode library does not support *_with_index type notation
+        #       Instead the index needs to be cached on the object itself
+        template.index
       end
-    end
+      register(header: 'Name') do |template|
+        template.id
+      end
+      file_header = "File (Dir: #{FlightJob.config.templates_dir})"
 
-    def self.build_output(**opts)
-      super(row_color: :cyan, header_color: :bold, **opts)
+      if verbose?
+        register_column(header: file_header) do |template|
```

Comment:
This is broke.  Do you mean `register`?

Reply:
73388a26fc3dafb4d3666b87ba91a014ef43768c